### DISCUSSION TITLE
Welcome to FastFlix Discussions!

### DISCUSSION THREAD

**Discussion by cdgriffith** [cdgriffith/FastFlix#423] [General]
## 👋 Welcome!
Discussions are new to FastFlix, but going to become the future standard for feature requests!

**Comment by ggrub** [comment]
Hi,

is it possible to make only one pass with hevc (nvenc) ?
i only need one pass encoding.

bye

**Reply by cdgriffith** [reply]
With the `ffmpeg` version not currently. However if you install the `rigaya` encoder, there is a `multipass` option that you can set to `None`
